dipy/viz/streamline.py

```diff
@@ -31,6 +31,14 @@ def show_bundles(bundles, interactive=True, view='sagital', colors=None,
         Default is None.
 
     """
+
+    if view == 'sagital':
+        warnings.warn(
+            f"The view argument value `{view}` is deprecated and its support will be "
+            f"removed in a future version. Please, use ``sagittal`` instead."
+        )
+        view = "sagittal"
```

Comment:
@skoudoro Have not found a more elegant way to do this as it is a parameter value.

Reply:
ok, fine